cloud-sql/r2dbc/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.3.4.RELEASE</version>
+		<relativePath/>
+	</parent>
+	<groupId>com.example.cloudsql</groupId>
+	<artifactId>r2dbc-sample</artifactId>
```

Comment:
Should it be `r2dbc-mysql-sample` because we might have an example for SQL Server and Postgres in the future?
The directory would need to b e renamed as well.

Reply:
@meltsufin  Actually it is going to work with postgres as well, if the right dependencies are added. Also the connection string needs to be updated. I'll modify the sample to mention that.

Reply:
done

Reply:
We would prefer 2 different samples, even if the code is 99% the same between them. This makes it easier for us to add region tags and integrate in as quickstarts. The project folder should be something like `cloud-sql/spring-boot-r2dbc/mysql`.



 I'm okay if you want to leave this PR as MySQL and my team can handle making changes for postgres and sqlserver if you prefer.